# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/dom/idlharness.https.html.ini
# Commit: 40fa8e4b06c8
# Full Hash: 40fa8e4b06c85e9e241da6e56b62a0b2bde7427c
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2024-03-28 17:08:58
# Regressor Bug: 1887729
# File Overlap Count: 2
# Description:
#   Bug 1887729 - Implement context lost/restored support for CanvasRenderingContext2D. r=webidl,gfx-reviewers,smaug,lsalzman
#   
#   Remote canvas can run in the GPU process, and if the GPU process
#   crashes, we need to notify the application using canvas. Historically we
#   just failed, and the application may have been able to continue drawing
# ==============================================================================

diff -r 469884ac63bc -r 40fa8e4b06c8 testing/web-platform/meta/html/dom/idlharness.https.html.ini
--- a/testing/web-platform/meta/html/dom/idlharness.https.html.ini	Thu Mar 28 01:16:16 2024 +0000
+++ b/testing/web-platform/meta/html/dom/idlharness.https.html.ini	Thu Mar 28 01:51:23 2024 +0000
@@ -282,21 +282,6 @@
   [ImageData interface: new ImageData(10, 10) must inherit property "colorSpace" with the proper type]
     expected: FAIL
 
-  [CanvasRenderingContext2D interface: operation isContextLost()]
-    expected: FAIL
-
-  [CanvasRenderingContext2D interface: document.createElement("canvas").getContext("2d") must inherit property "isContextLost()" with the proper type]
-    expected: FAIL
-
-  [OffscreenCanvasRenderingContext2D interface: operation isContextLost()]
-    expected: FAIL
-
-  [SVGElement interface: attribute oncontextlost]
-    expected: FAIL
-
-  [SVGElement interface: attribute oncontextrestored]
-    expected: FAIL
-
   [SVGElement interface: attribute onbeforematch]
     expected: FAIL
 
@@ -770,42 +755,6 @@
   [Window interface: attribute originAgentCluster]
     expected: FAIL
 
-  [Window interface: attribute oncontextlost]
-    expected: FAIL
-
-  [Window interface: attribute oncontextrestored]
-    expected: FAIL
-
-  [Window interface: window must inherit property "oncontextlost" with the proper type]
-    expected: FAIL
-
-  [Window interface: window must inherit property "oncontextrestored" with the proper type]
-    expected: FAIL
-
-  [Document interface: attribute oncontextlost]
-    expected: FAIL
-
-  [Document interface: attribute oncontextrestored]
-    expected: FAIL
-
-  [Document interface: iframe.contentDocument must inherit property "oncontextlost" with the proper type]
-    expected: FAIL
-
-  [Document interface: iframe.contentDocument must inherit property "oncontextrestored" with the proper type]
-    expected: FAIL
-
-  [Document interface: new Document() must inherit property "oncontextlost" with the proper type]
-    expected: FAIL
-
-  [Document interface: new Document() must inherit property "oncontextrestored" with the proper type]
-    expected: FAIL
-
-  [Document interface: documentWithHandlers must inherit property "oncontextlost" with the proper type]
-    expected: FAIL
-
-  [Document interface: documentWithHandlers must inherit property "oncontextrestored" with the proper type]
-    expected: FAIL
-
   [Window interface: attribute onbeforematch]
     expected: FAIL
 
@@ -892,18 +841,6 @@
   [HTMLMediaElement interface: document.createElement("audio") must inherit property "getStartDate()" with the proper type]
     expected: FAIL
 
-  [HTMLElement interface: attribute oncontextlost]
-    expected: FAIL
-
-  [HTMLElement interface: attribute oncontextrestored]
-    expected: FAIL
-
-  [HTMLElement interface: document.createElement("noscript") must inherit property "oncontextlost" with the proper type]
-    expected: FAIL
-
-  [HTMLElement interface: document.createElement("noscript") must inherit property "oncontextrestored" with the proper type]
-    expected: FAIL
-
   [HTMLLinkElement interface: attribute blocking]
     expected: FAIL
 